cli/chaosli/cmd/explain.go

```diff
@@ -290,4 +246,9 @@ func explanation(path string) {
 
 func init() {
 	explainCmd.Flags().String("path", "", "The path to the disruption file to be explained.")
+	err := explainCmd.MarkFlagRequired("path")
+
+	if err != nil {
```

Comment:
What kind of errors can happen here? Should a message be printed to the user?

Reply:
the error would be if the required path flag is not given. The MarkFlagRequired function sends out the error message on error with the help command as well, so it should be clear to the user what is wrong when that function errors.